client/lib/analytics/index.js

```diff
@@ -287,20 +283,27 @@ const analytics = {
 				return;
 			}
 
+			let actualProperties = {};
```

Comment:
Do we still need `actualProperties`? I find it confusing (the `event` vs `actual` usage), and from what I can tell, with the new `site` being passed to `getAll()`, I think `eventProperties` is all we need?

Reply:
Otherwise, I've gone through the list in the description and didn't find any blog IDs getting past (no site-specific super props at all). 👍